index-ecology.md

```diff
@@ -5,7 +5,9 @@ layout: galaxy
 {% include notices.html %}
 {% include maintenance.html %}
 
-Welcome to **Galaxy for Ecology**
+Welcome to **Galaxy for Ecology** -- a web platform to get, process, analyze and visualize ecological data
+
+![2018 Concarneau Galaxy-E team](https://github.com/65MO/Galaxy-E/blob/master/galaxy/static/Galaxy-E-concarneau-team-2018-logo.gif)
```

Comment:
Can you store the image locally under https://github.com/usegalaxy-eu/website/tree/master/assets/media?

Reply:
For sure!